docs/docs/05-database-fundamentals.md

```diff
@@ -71,3 +71,23 @@ RENAME COLUMN old_name TO new_name;
 ```sql
 TRUNCATE TABLE table_name;
 ```
+
+### [1.4] DCL (Data Control Language)
+เป็นกลุ่มคำสั่งที่จะช่วยให้ผู้บริหารฐานข้อมูล (DBA)
+สามารถควบคุมฐานข้อมูลเพื่อ<u>กำหนดสิทธิการอนุญาต</u> (Grant)  หรือ <u>การยกเลิกการเข้าใช้</u> (Revoke) ฐานข้อมูล ซึ่งเป็นกระบวนการป้องกันความปลอดภัยในฐานข้อมูล
+
+* GRANT: คำสั่งนี้<u>ให้สิทธิ์</u> users ในการเข้าถึงฐานข้อมูล
+
+```sql
+GRANT privileges_names ON object TO user;
+```
+
+* REVOKE: คำสั่งนี้จะ<u>ถอนสิทธิ์</u>การเข้าถึงของ users
+
+```sql
+REVOKE privileges_names ON object FROM user;
+```
+
+privileges_names ที่ GRANT หรือ REVOKE ได้
+
+!["fig2"](https://media.geeksforgeeks.org/wp-content/uploads/sqltable.jpg)
```

Comment:
I think we should create a table in Markdown. This way we don't have to worry about the copyright. You can try [Markdown Tables generator](https://www.tablesgenerator.com/markdown_tables).

Reply:
Great tool, Thank you